### ISSUE TITLE
stop automatically creating default identity and modify how default and proxy addresses are handled.

### ISSUE THREAD

**Issue by julonexus** [BelledonneCommunications/linphone-desktop#376]
From Gitlab by  "IT wrx":

"I think it's counterproductive/unintuitive to have linphone-desktop automatically create a default identity using the lan ip. IMHO, it would make more sense to just let the user create their sip accounts and have the option to mark one as default. Separating "default" vs "proxy" accounts in the GUI is also unintuitive for normal users, IMO. Instead, adding a "proxy address" field to account settings would keep things simpler in the more frequent/prominent parts of the GUI (e.g. settings window->manage sip accounts tab). Having the current "automatic default identity" results in an unused identity for, at least some, use cases with more cluttered/confusing GUI elements, IMO.

I think this issue was largely prompted by me not understanding how to switch profiles when testing before. I just installed on a fresh machine and the set up went much smoother. One thing that might make things easier is to have tooltips on hover. If the active profile icon said "click to change active profile" when hovering over it, people could more easily discover how to do that. Though, i still think it makes sense to use the account they set up, as the default from the beginning, and maybe don't create a local account for them. Or maybe just naming it local account and having it not be the default would be the best of both worlds."

**Comment by UltimateByte** [comment]
I agree this is super annoying and a quick googling confirms that.
Also it prompts for a login for the default identity on startup since it fails to connect to nothing, which makes useless clicks every single time.

**Comment by vince62s** [comment]
I fully agree too. On top of this it creates a mismatch between incoming calls logs and the "active sip account". All incoming missed calls get allocated to the default inedity and only visible when we force the default identity username to the same as the username of the sip account.
Please fix all of this. Cheers.

**Comment by tglemser** [comment]
There is a new option in the current alpha https://www.linphone.org/news/ldap-search-feature-available-linphone-desktop to hide the local account. Add the following to your linphonerc (also 

```
[ui]
show_local_sip_account=0 

```
If you are using the XML-Configuration-Feature for provisioning, ad this to your XML-Files
```
<section name="ui">
<entry name="show_local_sip_account">0</entry>
</section>

```

**Comment by clementmoulin** [comment]
Using version 4.4.21, and after adding:
```
[ui]
show_local_sip_account=0
```
Default identity is still present...

**Comment by olku** [comment]
I wholeheartedly agree. This default lan ip identity is annoying, leads to misunderstandings, and quite frankly is useless.

**Comment by tglemser** [comment]
Interesting. I'm running Windowsdesktop 4.3.1. The only account being shown is my configured SIP-ID having show_local_sip_account=0 in the linphonerc in section [ui]

**Comment by ant0nwax** [comment]
I agree that the default account is not necessary for most of us who use SIP accounts and not an Asterix... I cannot hide the default id with the recommended addition to linphonerc

i added to ~/linphonerc
```
[ui]
show_local_sip_account=0
```
after adding I restarted linphone

did not remove the default identity on Ubuntu 22.04 LTS
despite of that i lost all account settings in the latest Ubuntu System Update :) but that is off topic

**Comment by anourthebrit** [comment]
Would just like to add that I still get this issue, in fact when I set show_local_sip_account=0 in linphonerc and close the app and reopen it, that setting has gone from the file. As far as we can see this linphonerc is being overwritten on close and reopen by something else but I cannot find where that something else is and/or how to stop it.

**Comment by prjcarr** [comment]
> Would just like to add that I still get this issue, in fact when I set show_local_sip_account=0 in linphonerc and close the app and reopen it, that setting has gone from the file. As far as we can see this linphonerc is being overwritten on close and reopen by something else but I cannot find where that something else is and/or how to stop it. Mac OS, Desktop App download (not self built)

Try adding it with the app closed first.

**Comment by anourthebrit** [comment]
> > Would just like to add that I still get this issue, in fact when I set show_local_sip_account=0 in linphonerc and close the app and reopen it, that setting has gone from the file. As far as we can see this linphonerc is being overwritten on close and reopen by something else but I cannot find where that something else is and/or how to stop it. Mac OS, Desktop App download (not self built)
> 
> Try adding it with the app closed first.

Thanks @prjcarr, weve tried adding it with the app closed too. we've run a trace, and can see liblinphone creating a temporary file on start up, and some other events that creates a temporary linphonerc file with some set defaults that overwrites the linphonerc config every time.
